### DISCUSSION TITLE
How to debug C++ projected in C# using CsWinRT?

### DISCUSSION THREAD

**Discussion by OmarBazaraa** [microsoft/CsWinRT#1525] [Q&A] [answered]
I'm following the [Simple Math Component](https://learn.microsoft.com/en-us/windows/apps/develop/platform/csharp-winrt/net-projection-from-cppwinrt-component) tutorial for projecting and running C++/WinRT from within C# projects using CsWinRT, and I want to debug into the C++ code when running the C# application. The problem is that the C++-generated DLL is loaded by WinRT at runtime and I cannot find it in the debugging modules (and I cannot load its symbols since the DLL is not loaded by Visual Studio).
Is there anyway to workaround this to be able to debug the C++ code when running the C# projections?

**Comment by lhak** [comment, marked as answer]
Enabling mixed mode debugging in the visual studio settings will allow you to do this

**Reply by OmarBazaraa** [reply]
Thanks a lot @lhak! This actually worked :)
